system/hardware/tici/hardware.h

```diff
@@ -103,4 +103,9 @@ class HardwareTici : public HardwareNone {
 
   static bool get_ssh_enabled() { return Params().getBool("SshEnabled"); }
   static void set_ssh_enabled(bool enabled) { Params().putBool("SshEnabled", enabled); }
+
+  static void config_cpu_rendering() {
```

Comment:
any way to assert that the mesa GL is loaded?

Reply:
probably not here since we don't have the GL context, but within mapsd we probably can with

```glGetString​(GL_RENDERER)``` or ```glGetString(GL_VENDOR)```